cluster/src/main/java/org/apache/iotdb/cluster/metadata/CMManager.java

```diff
@@ -1565,7 +1565,7 @@ public MNode getMNode(MNode deviceMNode, String measurementName) {
               () -> {
                 try {
                   showTimeseries(group, plan, resultSet, context);
-                } catch (CheckConsistencyException e) {
+                } catch (CheckConsistencyException | MetadataException e) {
```

Comment:
The show timeseries root... command is used to query all DataGroup groups. If a DataGroup group does not have a sequence, an error is reported. As a result, other DataGroups that have a sequence cannot obtain the result.